modules/order/src/Entity/Order.php

```diff
@@ -469,6 +519,18 @@ public static function baseFieldDefinitions(EntityTypeInterface $entity_type) {
       ->setDisplayConfigurable('view', TRUE)
       ->setSetting('workflow_callback', ['\Drupal\commerce_order\Entity\Order', 'getWorkflowId']);
 
+    $fields['adjustments'] = BaseFieldDefinition::create('commerce_order_adjustment')
+      ->setLabel(t('Adjustments'))
+      ->setDescription(t('Adjustments to the order total.'))
+      ->setRequired(FALSE)
+      ->setCardinality(BaseFieldDefinition::CARDINALITY_UNLIMITED)
+      ->setDisplayOptions('form', [
```

Comment:
Crud, we actually _don't_ want to expose the UI for this, line item adjustments are added via backend processes - like manual code or promotions sytem.